geowebcache/rest/src/main/java/org/geowebcache/rest/controller/GWCIndexController.java

```diff
@@ -35,7 +35,7 @@
 @RequestMapping(path="${gwc.context.suffix:}/rest")
 public class GWCIndexController {
 
-    @RequestMapping(value = "/about", method = RequestMethod.GET, produces = "text/html")
+    @RequestMapping(method = RequestMethod.GET, produces = "text/html")
```

Comment:
I assume this is the root of the rest API ? Does it also need to be available from `/about`, in which case we would want `value = {"", "/about"}` ?

Reply:
"About" was a bad endpoint, the former IndexRestlet responded on the /rest endpoint. This may also be contributing to the GS root rest endpoint not working.